r/R/json.R

```diff
@@ -20,6 +20,8 @@
 #' Using [JsonTableReader]
 #'
 #' @inheritParams read_delim_arrow
+#' @param schema [Schema] that describes the table.  If provided, it will be
+#' used to satisfy both `col_names` and `col_types`.
```

Comment:
If this were the right docs to include here, you wouldn't need to add them--`@inheritParams read_delim_arrow` would pull it in for you. But since there are no `col_names` and `col_types` arguments to this function, we should specify something different (which will override what inheritParams brings in).

```suggestion
#' @param schema [Schema] that describes the column names and data types in the table.
```